variants

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 Akamai Technologies
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file specifies the images that are built by this package.
+#
+# Each line describes an image that is pushed to dockerhub.
+# The last word of each line determines the tag, e.g. appsec -> akamai/appsec.
+
+# base: shared base image for all variants
+# DO NOT DELETE this line
+base
+
+# cli: base + akamai cli wrapper
+# DO NOT DELETE this line
+base cli
+
+# specialized variants
+cli adaptive-acceleration
+cli api-gateway
+cli appsec
+cli cloudlets
+cli cps
+cli dns
+cli edgeworkers
+cli firewall
+cli image-manager
+cli property-manager
+cli property
+cli purge
+cli sandbox
+cli visitor-prioritization
```

Comment:
Do we want to add the cli terraform package https://github.com/akamai/cli-terraform or include it inside the terraform image?

Reply:
good question! I'd forgotten about that CLI. @lukaszczerpak thoughts?

Reply:
it makes sense to create a new image variant with terraform + terraform-cli

Reply:
referenced in #14